### DISCUSSION TITLE
TaskCanvasTabs : Tab title does not display

### DISCUSSION THREAD

**Discussion by hawkng** [twilio-professional-services/flex-project-template#129] [Q&A] [answered]
Is icon mandatory?

`  <Tab label='Video' icon="accept">
      <Flex vertical width={"100%"}>
         <Box width="100%" position={"absolute"} left={0}>
           <Flex vertical hAlignContent="center" width={"100%"}>
    ...
           </Flex>
         </Box>
       </Flex>
    </Tab>`
    
![image](https://user-images.githubusercontent.com/29279065/211002394-ab73344f-ccd9-4ba7-8c6f-3d4a462f19c2.png)

**Comment by dremin** [comment, marked as answer]
`icon` is not required on the `Tab` component, omitting it will allow the `label` to be shown instead. I believe this prop may have been mistakenly marked as required in the past, but that has since been resolved in more recent Flex UI versions.
